project-base/app/docker/php-fpm/Dockerfile

```diff
@@ -57,6 +57,11 @@ RUN if [ -n "${www_data_gid}" ] && ! getent group ${www_data_gid} >/dev/null; th
         if id -u www-data >/dev/null; then usermod -u ${www_data_uid} www-data; fi; \
     fi
 
+RUN mkdir -p ./vendor \
+    && mkdir -p ${project_root}/vendor \
```

Comment:
Do we need both vendor folders? 
Both variants (monorepo / project-base run) should have the vendor folder at the root (/var/www/html), or?